.github/workflows/upmerge.yaml

```diff
@@ -1,3 +1,18 @@
+# This workflow automates the process of upmerging changes from the current release branch to the edge branch.
+# During the course of a release, the release branch is the default branch so that PRs can be immediately
+# brought into the release without waiting for a new release. This workflow merges those changes into 
+# the edge branch so that edge can be used as the basis for the next release branch.
+#
+# It is triggered manually via the workflow_dispatch event.
+#
+# The workflow performs the following steps:
+# 1. Checks out the edge branch.
+# 2. Configures git with a user name and email.
+# 3. Creates a new branch from edge.
+# 4. Merges changes from the source branch into the new branch.
```

Comment:
Is the source branch **edge** or the **release branch**? Because if it is **edge** and if we create a branch from **edge**, the new branch will not have any changes.

The documentation is great by the way. Thanks! Can we also add an example like main branch is 0.36, edge, new branch...

Reply:
✅ I added an example and clarification about which branch is the release branch.